File: Backend/EcoMonitor.Contracts/Contracts/BinPhotoResponse.cs
namespace EcoMonitor.Contracts.Contracts
{
    public record BinPhotoResponse(
       Guid Id,
       string FileName,
       string UrlFile,
       double Latitude,
       double Longitude,
       DateTime UploadedAt,
       string BinType,
       double FillLevel,
       bool IsOutsideBin,
       string Comment);
}

File: Frontend/src/types/BinPhotoResponse.ts
export interface BinPhotoResponse {
    id?: string;
    fileName: string;
    urlFile: string;
    latitude: number;
    longitude: number;
    uploadedAt: Date;
    binType: string;
    fillLevel: number;
    IsOutsideBin: boolean;
    comment: string;
}